tests/server/testServerWithStorage.js

```diff
@@ -327,23 +376,49 @@ sjrk.storyTelling.server.testServerWithStorageDefs = [{
     },
     // Story with an image
     {
+        // initializes the story in its unpublished form
```

Comment:
Line 373-616 is a very long sequence. It should be broken down to a few sequences, each testing a meaning feature.

Reply:
I agree that it's a little bit spaghetti, with each test feeding into the next. Breaking it into sequences would also allow for more variety in the test cases.

Doing this will take a fair amount of time, I believe, so we may want to revisit the release date in that case.